src/lib/work_partitioner/work_partitioner.ml

```diff
@@ -399,7 +440,7 @@ let submit_partitioned_work ~(result : Work.Result.Partitioned.Stable.Latest.t)
       ; data = { proof; data = elapsed }
       } ->
       let submitted_result = Work.Result.Single.Poly.{ spec; proof; elapsed } in
-      submit_single ~partitioner ~submitted_result ~job_id
+      submit_single ~partitioner ~submitted_result ~job_id ()
```

Comment:
So in all, this invocation is all evil of one bug I discovered when fixing nightly.